src/clib/u8g_com_samd51_hw_spi.cpp

```diff
@@ -58,14 +58,7 @@ uint8_t u8g_com_samd51_hw_spi_fn(u8g_t *u8g, uint8_t msg, uint8_t arg_val, void
       break;
 
     case U8G_COM_MSG_CHIP_SELECT:         // arg_val == 0 means HIGH level of U8G_PI_CS
-      if (arg_val == 0) {
-        // disable
-        u8g_SetPILevel(u8g, U8G_PI_CS, HIGH);
-      }
-      else {
-        // enable
-        u8g_SetPILevel(u8g, U8G_PI_CS, LOW);
-      }
+      u8g_SetPILevel(u8g, U8G_PI_CS, arg_val ? LOW : HIGH);
```

Comment:
I think I made an error here. CS is active high in other implementations (comment in case has to be removed)...I'll post a fix to it

Reply:
It matches `stm32duino_hw_spi`, so it is could be correct.